edgexfoundry/device-usb-camera
# fix: support external RTSP servers using RtspServerMode

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

https://github.com/edgexfoundry/edgex-docs/pull/1203

## Testing Instructions
1. run `make test`
2. verify all tests pass
3. start up edgex via edgex-compose with no-secty: `make run no-secty`
4. run device-usb-camera with `disableRtspServer` set to false `./device-usb-camera -cp -d -o`
5. verify RTSP server starts
6. run device-usb-camera with `disableRtspServer` set to true `./device-usb-camera -cp -d -o`
7. verify RTSP server does not start


## New Dependency Instructions (If applicable)
